src/main/java/com/pmm/budgetcalculator/expensing/domain/expense/ExpenseService.java

```diff
@@ -0,0 +1,35 @@
+package com.pmm.budgetcalculator.expensing.domain.expense;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class ExpenseService {
+
+    private final ExpenseRepository expenseRepository;
+
+    public List<Expense> getAllExpenses() {
+        return expenseRepository.findAll();
+    }
+
+    public Expense getExpenseById(Long id) {
+        return expenseRepository.findById(id).orElseThrow(() -> new RuntimeException("Expense not found for Id:" + id));
```

Comment:
For the sake of transparency: when you call multiple methods one by one put each in new line starting with dot:                             
expenseRepository                                   
    .findById(id)                                      
     .orElseThrow();                                    
Writing code is easy - reading is the hard part :)